storage/quota/grpquota.go

```diff
@@ -59,20 +60,22 @@ func (quota *GrpQuotaDriver) EnforceQuota(dir string) (string, error) {
 
 	mountPoint, hasQuota, _ := quota.CheckMountpoint(devID)
 	if len(mountPoint) == 0 {
-		return mountPoint, fmt.Errorf("mountPoint not found for the device on which dir %s lies", dir)
+		return mountPoint, fmt.Errorf("failed to find mountpoint: (%s)", dir)
 	}
 	if !hasQuota {
 		// remount option grpquota for mountpoint
-		// FIXME: ignore or record err? Why inconsistent with prjquota (rudyfly)
-		_, _, _, err := exec.Run(0, "mount", "-o", "remount,grpquota", mountPoint)
+		exit, stdout, stderr, err := exec.Run(0, "mount", "-o", "remount,grpquota", mountPoint)
 		if err != nil {
-			return "", err
+			logrus.Errorf("failed to remount grpquota, mountpoint: (%s), stdout: (%s), stderr: (%s), exit: (%d), err: (%v)",
```

Comment:
maybe errMsg :=  fmt.Sprintf("failed to remount grpquota, mountpoint: (%s), stdout: (%s), stderr: (%s), exit: (%d),  err: (%v)", ...)